src/pages/RestaurantAdminView/RestaurantAdminView.jsx

```diff
@@ -11,7 +22,37 @@ import EditDetailsModal from '../../components/EditDetailsModal/EditDetailsModal
 import languageSelector from '../../assets/languages/languageSelector';
 
 const RestaurantAdminView = () => {
-  const usersData = userDB;
+  const user = useSelector(state => state.userReducer.user);
+  const loading = useSelector((state) => state.userReducer.loading);
+  const language = useSelector(state=> state.languageReducer);
+  const [restaurant, setRestaurant] = useState(null);
+  const cookies = new Cookies();
+  const dispatch = useDispatch();
+  const jwtToken = cookies.get('token');
+  const { isExpired } = useJwt(cookies.get("token"));
+  const config = {
+    headers: {
+      Authorization: `Bearer ${jwtToken}`,
+    },
+  };
+  
+  useEffect(() => {
+    const fetchData = async () => {
+      dispatch(fetchUserRequest());
+      try {
+        const userResponse = axios.get(`${API_URL}/users/profile`, config);
+        const restaurantResponse = axios.get(`${API_URL}/restaurants/byuser/${user.id}`, config);
```

Comment:
`/restaurants/by-user`

Reply:
This component is being worked on by Nicolas at the moment, so I prefer not to modify his code at this time.

Reply:
ok, no problem, let's move it to approve